instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/test/groovy/S3TracingTest.groovy

```diff
@@ -237,7 +237,7 @@ class S3TracingTest extends AgentInstrumentationSpecification {
             "http.method" "POST"
             "http.status_code" 200
             "http.url" String
-            "http.user_agent" String
+            "$SemanticAttributes.USER_AGENT_ORIGINAL" String
```

Comment:
nit: 
```suggestion
            "user_agent.original" String
```
(just reads more consistently with the others). Also on line 703 and a few other test files.

Reply:
ideally they would all go in the other direction and use the constants